3-data-sharing-exploration/README.md

```diff
@@ -521,6 +536,15 @@ The used subscription must have the Microsoft.Purview provider registered.
 The used subscription must have the Microsoft.EventHub and Microsoft.Storage
 registered.
 
+### Missing ODBC Driver for SQL Server
```

Comment:
Good catch! Actually as we are changing it, could you please add this pre-requisite to the pre-requirement list for the non-container version?

Reply:
Will do